db/migrate/20160415121724_create_cloud_services.rb

```diff
@@ -0,0 +1,21 @@
+class CreateCloudServices < ActiveRecord::Migration[5.0]
+  def change
+    create_table :cloud_services do |t|
+      t.string :ems_ref
+      t.string :source
+      t.string :binary
+      t.string :host_name
```

Comment:
This comes from https://github.com/fog/fog-openstack/blob/master/lib/fog/openstack/models/compute/service.rb#L10 . It is the name of the host as Fog got it from OpenStack Nova Services list - identifier of Host in OpenStack.